src/pybind/mgr/cephadm/module.py

```diff
@@ -1490,6 +1490,21 @@ def get_hosts(self):
         """
         return list(self.inventory.all_specs())
 
+    @handle_orch_error
+    def get_gather_facts(self, hostname: Optional[str] = None) -> List[Dict[str, Any]]:
```

Comment:
I'd suggest `get_facts()` instead because:
* `get_gather_facts` is a bit redundant
* `gather_facts` implies that you are collecting the facts from the nodes, but that was already done: you're simply getting the facts from a cache.